hangups/auth.py

```diff
@@ -20,7 +20,7 @@
 import mechanicalsoup
 import requests
 
-from hangups import version
+from . import version
```

Comment:
ok, reverted in https://github.com/tdryer/hangups/pull/343/commits/5b290eadbfc5a42d3c0168730e7cf06d94e6c51d